helm_deploy/offender-search/templates/_envs.tpl

```diff
@@ -22,4 +25,27 @@ env:
   - name: AWS_REGION
     value: "eu-west-2"
 
+  - name: ELASTICSEARCH_PORT
+    value: "{{ .Values.elasticSearch.port }}"
+
+  - name: ELASTICSEARCH_HOST
+    value: "{{ .Values.elasticSearch.host }}"
+
+  - name: ELASTICSEARCH_SCHEME
+    value: "{{ .Values.elasticSearch.scheme }}"
+
+  - name: AWS_ROLEARN
+    valueFrom:
+      secretKeyRef:
+        name: offender-search-delius-elastic-search-secret
```

Comment:
Curious, how does the secret get into the namespace? Is it from the Cloud Platform Terraform?

Reply:
yep: https://github.com/ministryofjustice/cloud-platform-infrastructure/blob/master/terraform/cross-account-IAM/offender-search-dev.tf#L31-L42